src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
Yes, This helps to release control from developer to do CSS coding for their application.